skyvern/forge/sdk/routes/webhooks.py

```
@@ -0,0 +1,133 @@
+    include_in_schema=False,
+)
+@base_router.get(
+    "/internal/runs/{run_id}/test-webhook",
+    tags=["Internal"],
+    response_model=RunWebhookPreviewResponse,
+    summary="Preview webhook replay payload",
+    include_in_schema=False,
+)
+async def preview_webhook_replay(
+    run_id: str,
+    current_org: Organization = Depends(org_auth_service.get_current_org),
```

Ruff (0.14.1)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)